### ISSUE TITLE
Null handling

### ISSUE THREAD

**Issue by YurySolovyov** [roman01la/imtbl#2]
Is that intentional that `null` values are not being handled the same way as `undefined`?

```js
assocIn({ a: null }, ['a', 'b'], 1)
// -> { a: null }
assocIn({ a: undefined }, ['a', 'b'], 1)
// -> { a: { b: 1 } }
```

**Comment by roman01la** [comment]
It's definitely an issue. Feel free to submit a fix. Thanks.

**Comment by roman01la** [comment]
fixed in #3
